libkbfs/data_types.go

```diff
@@ -640,6 +641,28 @@ func (et EntryType) String() string {
 	return "<invalid EntryType>"
 }
 
+// EXCL indicates whether O_EXCL is set on a fuse call
+type EXCL uint32
+
+const (
+	// NoEXCL indicates O_EXCL is not set
+	NoEXCL EXCL = 0
+
+	// WithEXCL indicates O_EXCL is set
+	WithEXCL EXCL = syscall.O_EXCL
```

Comment:
This is a neat trick, but I wonder if it will make the eventual Windows implementation trickier.  Maybe just use `iota` for these and add a simple function to make an `EXCL` from a syscall uint32?